dev/io.openliberty.netty.internal/src/io/openliberty/netty/internal/ConfigConstants.java

```diff
@@ -16,9 +16,11 @@
 
 public interface ConfigConstants {
 
-    public AttributeKey<String> NameKey = AttributeKey.valueOf("Name");
-    public AttributeKey<String> HostKey = AttributeKey.valueOf("Host");
-    public AttributeKey<String> ExternalHostKey = AttributeKey.valueOf("ExternalHost");
-    public AttributeKey<Integer> PortKey = AttributeKey.valueOf("Port");
+    public final String EXTERNAL_NAME = "ExternalName";
+    public final AttributeKey<String> NAME_KEY = AttributeKey.valueOf(EXTERNAL_NAME);
```

Comment:
Looks odd just having one of these strings and not having EXTERNAL_NAME_KEY but NAME_KEY  when there is EXTERNAL_HOST_KEY ?

Reply:
It looks odd indeed, I believe it was because it is used in another piece of code. I'm updating to add string keys for all the other attributes as well to make it consistent.